docs/disk.rst

```diff
@@ -1,7 +1,10 @@
 Disk Format
 ===========
 
-ethPM cli writes package assets to your disk using the following format.
+_ethpm_packages/
+----------------
+
+ethPM cli writes ethPM package assets to your disk using the following format. You can have multiple different ``_ethpm_packages/`` local directories. The ``--ethpm-dir`` flag is available on most ``ethpm-cli`` commands to target a specific ``_ethpm_packages/`` dir. You can also set the environment variable ``ETHPM_CLI_PACKAGES_DIR`` to your target ``_ethpm_packages/`` directory. Otherwise, the cli will automatically lookup any ``_ethpm_packages/`` directory available under the current working dirctory.
```

Comment:
This is probably better laid out so that it's clear what order of precedence these will go in.  Something like this.

- by default it will write files to `./_ethpm_packages/` in the current working directory.
- If `--ethpm-dir` is provided then the CLI will use this.
- If the environment variable `ETHPM_CLI_PACKAGES_DIR` is set the CLI will use this if one is not specified using the `--ethpm-dir` flag